### PR TITLE
[HPT -401] Bug Fix: Adds button to view in bookreader back to paged display.

### PR DESCRIPTION
**Pull request by andjsmit**
Also fixes paged display error if no pages added to paged object and fixes breadcrumb on paged edit page.

### CODE REVIEW

File: app/views/pageds/show.html.erb [hunk 1]
```diff
@@ -26,16 +26,20 @@
...
+    <div>
+      <%= button_to 'Update', edit_paged_path(@paged), :method => :get %>
+    </div>
   </div>
   
   <!-- Cover Image -->
-  <div style="float:left; padding-left:50px;">
-    <%= image_tag(@ordered[0]["ds_url"], height: "64", width: "64" ) %>
+  <div style="float:left; padding-left:50px; text-align:center;">
+    <% if @ordered[0] %>
```
Comment (mwoodiupui) [inline app/views/pageds/show.html.erb:36]: We ought to have a functional test for this.  I was surprised by the same issue when manually testing another patch.  (See https://bugs.dlib.indiana.edu/browse/HPT-411)

**Comment by andjsmit** [comment]
Feature test added for paged display without pages.

**Comment by mwoodiupui** [comment]
I cherrypicked these commits to my own feature branch, to facilitate manual testing, and they seem to work well.  +1

**Comment by andjsmit** [comment]
Converted "View in BookReader" button to a link surrounded by a button tag so that a target of _blank could be added/reutrned to the link.

**Pull request merged**